### DISCUSSION TITLE
Add "Follow @ap.brid.gy" on the bridged Bluesky profile and vice-versa

### DISCUSSION THREAD

**Discussion by sol2070** [snarfed/bridgy-fed#1347] [Ideas]
The description appended to a bridged Bluesky profile is
```
[bridged from instance.social/@user on the fediverse by fed.brid.gy]
```
It would be useful to add something like "Follow @ap.brid.gy to interact with this profile".
The same for a bridged Fediverse profile.
Because interactions would not be seen on the other side, and the person interacting doesn't know this.

**Comment by snarfed** [comment]
I'd love to add more info and instructions! Bluesky profile bios are pretty short though, just 256 characters max, so I'm reluctant to take any more than I have to away from the user's own bio. Hopefully the link to https://fed.brid.gy/ helps!...since it has loud instructions on how to get started, eg following @ap.brid.gy.

**Reply by sol2070** [reply]
I see. Maybe the language could be a bit more informative. For me, it's totally fine. But Bluesky users usually don't get it. Specially because there's no message informing them that the interactions with the bridged profile will not be seen, unless they follow the bridge account. 
Just my thoughts. Anyway, thanks for this wonderful piece of software!

**Reply by FurblandChannel** [reply]
> I see. Maybe the language could be a bit more informative. For me, it's totally fine. But Bluesky users usually don't get it. Specially because there's no message informing them that the interactions with the bridged profile will not be seen, unless they follow the bridge account. Just my thoughts. Anyway, thanks for this wonderful piece of software!

Perhaps a pinned post of some sort?

**Reply by TomCasavant** [reply]
I think a pinned post is a good idea, #1374 it sounds like Bluesky only supports a single pinned post right now- it might be easier short term to just pin a informative post so bridgy doesn't have to decide which pinned post to pin from the fediverse? 

I think you'd have to make it clear that the post is from bridgy and not from the AP user though

**Reply by FurblandChannel** [reply]
> I think a pinned post is a good idea, #1374 it sounds like Bluesky only supports a single pinned post right now- it might be easier short term to just pin a informative post so bridgy doesn't have to decide which pinned post to pin from the fediverse?
> 
> I think you'd have to make it clear that the post is from bridgy and not from the AP user though

That sounds like a good idea, perhaps just some small text that says something along the lines of “automatically generated by bridgy fed” at the bottom?

**Reply by Tamschi** [reply]
There's a related issue here about automatically sending a DM/Chat message to new followers: #1340

**Reply by snarfed** [reply]
Pinned post is an interesting idea! We already want to bridge the user's pinned posts too though, https://github.com/snarfed/bridgy-fed/issues/1374 , which would conflict.

Bridged profiles definitely need some explanation that they're bridged, and some way to learn more about that. In general though, I think I want the bridged profile to represent the user themselves as closely as possible, as opposed to advertising and explaining Bridgy Fed more loudly. A pinned post would definitely give us more real estate, but it would also detract from the user's own presence. I know the current footer and links aren't huge, but hopefully they're still easy enough to click through and see what Bridgy Fed is and how to use it.

**Reply by Pottit** [reply]
I think a direct encouragement to follow @ap.brid.gy would be more useful than the link to https://fed.brid.gy/, for a couple of reasons.

First, I would guess many users are more likely to check out another Bluesky profile than they are to read up on information on an external website. Leaving the platform/app seems like a big step. 

Second, linking to an external website gives the impression that the user actually needs to understand anything or make any sort of effort. It seems like not many people are willing to make an effort for anything these days, and we certainly shouldn't expect people to understand what bridging their account actually _means_.

I think something as short as  **"follow @ap.brid.gy to interact with this user"** would be enough information for user profiles. Those who want to learn more can find the relevant information from Bridgy Fed's profile, those who want to learn as little as possible can follow instructions and continue living their lives blissfully unaware while nevertheless being bridged. 

I don't know what the PM being sent to BS users with bridging requests currently contains, but a similar logic might apply. I have so far not had any success getting anyone to bridge - even among users who have signed up for Mastodon in the past. I remember there was also talk about possibly bridging all BS users at some point. I would certainly be in favour, but I guess I am biased. 


Edit:
If an URL to the original profile is considered important, this might be a better short form:
"[bridged from [url]; follow @ap.brid.gy to interact]"
With the url linking the original fediverse profile of the user. This way the user URL is maintained, while using as many characters as the current footnote.
